knowledge-base/how-to-render-any-html-tags-in-reports.md

```diff
@@ -0,0 +1,81 @@
+---
+title: How to render any HTML tags and CSS attributes in reports
+description: How to render any HTML tags and CSS attributes in reports.
+type: how-to
+page_title: How to render any HTML tags and CSS attributes in reports
+slug: how-to-render-any-html-tags-in-reports
+res_type: kb
+---
+
+## Description
+
+A frequently asked question is *how to render HTML tags and CSS attributes that are not supported by [HtmlTextBox item](../report-items-html-text-box)* in reports.  
+  
+## Solution  
+  
+The approach includes utilizing a [WebBrowser](https://docs.microsoft.com/en-us/dotnet/api/system.windows.forms.webbrowser?redirectedfrom=MSDN&view=net-5.0) control to render HTML to a bitmap with a help of a [user defined function](../expressions-user-functions) and a [PictureBox item](../report-items-picture-box) that will display the generated image (PNG):  
+
```

Comment:
Where does this code snippet goes? In which file, in which place, etc. This information is essential when providing code snippets.

Also think if it will make it more clear formatting the solution as a step by step guidance - using bullets